src/main.rs

```diff
@@ -18,31 +23,47 @@ lazy_static! {
 
 // static client: Client = Client::new("url", Auth::UserPass("user".to_owned(), "password".to_owned())).unwrap();
 
-// TODO: Task 1
+// Task 1
 fn time_to_mine(block_height: u64) -> Duration {
```

Comment:
Using unwrap is not recommended or safe.
If you are trying to return a Result then different functions return Result with different Error.
To unify then try using [Boxed Error trait](https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/boxing_errors.html)

Reply:
Hmm that's a good idea!! Also I was thinking of wrapping them into my [own](https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/wrap_error.html) error type